hack/build/run-functional-tests.sh

```diff
@@ -37,4 +37,9 @@ arg_oc="${KUBECTL:+-oc-path=$KUBECTL}"
 
 test_args="${test_args} -ginkgo.v ${arg_master} ${arg_namespace} ${arg_kubeconfig} ${arg_kubectl} ${arg_oc}"
 
+echo 'Wait until all CDI Pods are ready'
+until [[ $(./cluster/kubectl.sh get deployment cdi-file-host -n kube-system -o jsonpath={.status.availableReplicas}) -eq 1 ]] && [[ $(./cluster/kubectl.sh get deployment cdi-deployment -n kube-system -o jsonpath={.status.availableReplicas}) -eq 1 ]]; do
```

Comment:
we need to check that all pods in the install namespace are Running and "Ready". This insures that things like the http fileserver or anything else we add is up before we run the tests.

Reply:
I haven't found an easy way of determining if a Pod is 'Ready'. Which is why I went looking at the availableReplicas of the deployments instead of the pods.

Reply:
check this out https://github.com/kubevirt/kubevirt/blob/master/automation/test.sh#L138

Reply:
thanks